lametro/views.py

```diff
@@ -282,7 +282,10 @@ def get_context_data(self, **kwargs):
         end_date_str   = self.request.GET.get('to')
         day_grouper    = lambda x: (x.local_start_time.year, x.local_start_time.month, x.local_start_time.day)
 
-        minutes_queryset = EventDocument.objects.filter(note__icontains='minutes')
+        # We only want to display approved minutes
+        minutes_queryset = EventDocument.objects.select_related('event')\
```

Comment:
`select_related` is helpful if you need to access the related objects outside of your query, e.g., if we were going to iterate over these documents, accessing the related event in each loop. Here, Django does the join for you when you add the filter on the event extras. So, you can remove this `select_related` – unless I'm missing something?

Reply:
No, you're right!